ServiceWebsite/ServiceWebsite.Services/HearingsService.cs

```diff
@@ -43,7 +43,40 @@ private static bool HearingContainsParticipant(HearingDetailsResponse response,
         {
             return response.Participants.Any(p => p.Username.Equals(username, StringComparison.OrdinalIgnoreCase));
         }
-        
+
+        public async Task<Guid> GetParticipantId(string username, Guid id)
```

Comment:
Since the method is called `GetParticipantId` maybe renaming the `id` parameter to `hearingId` would make it clearer what kind of id it accepts?

Also, all async methods should be called `...Async` nowadays

Reply:
Resolved